.buildkite/cache-builder.yml

```
@@ -0,0 +1,23 @@
+# This pipeline runs via Buildkite's scheduled jobs feature.
+#
+# It's meant to rebuild various CI caches on a periodic async basis, so as
+# not to waste time on every CI job updating the cache.
+
+common_params:
```

💄  Nitpick: doesn't really matter, but since the `pipeline.yml` file uses `x-common-params` as the dummy key used to contain all our anchors we want to use later, it can make sense to make both keys consistent (either by using `x-common-params` in this new `cache-builder.yml`, or by using `common_params` key in `pipeline.yml`, it doesn't really matter which as long as we're consistent) 🙃

---

Oh, good point. I hadn't noticed that. Consistency FTW.

```suggestion
x-common-params:
```